javascript/ql/src/semmle/javascript/security/performance/ReDoSUtil.qll

```diff
@@ -112,7 +112,9 @@ class RegExpRoot extends RegExpTerm {
     // there are no lookbehinds
     not exists(RegExpLookbehind lbh | getRoot(lbh) = this) and
     // is actually used as a RegExp
-    isUsedAsRegExp()
+    isUsedAsRegExp() and
+    // is not inside a minified file.
```

Comment:
```suggestion
    // pragmatic performance optimization: ignore minified files.
```
All other queries that use `isMinified` have a syntactic reason for doing that, but in this case it is just an optimization, so we should document that.